internal/telemetry/cluster_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
similarly to the other one, can you tidy up this import block with

1. internal
2. own repo
3. third party
4. test if applicable

There's a linter that can do this for us if we want to pay attention to this: https://github.com/daixiang0/gci

This is not super important, functionality isn't impacted, it would just look more tidy 🙂